extensions/cornerstone/src/initWADOImageLoader.js

```diff
@@ -54,20 +54,39 @@ export default function initWADOImageLoader(
       // we should set this flag to false.
       convertFloatPixelDataToInt: false,
     },
-    beforeSend: function(xhr) {
+    beforeSend: function (xhr) {
       const headers = userAuthenticationService.getAuthorizationHeader();
 
       // Request:
       // JPEG-LS Lossless (1.2.840.10008.1.2.4.80) if available, otherwise accept
       // whatever transfer-syntax the origin server provides.
       // For now we use image/jls and image/x-jls because some servers still use the old type
       // http://dicom.nema.org/medical/dicom/current/output/html/part18.html
+
+      //Initialize the accept header content
+      const acceptHeader = ['multipart/related']
+      //Storing the requesting transfer syntax specified in OHIF config
+      const requestTransferSyntaxUID = appConfig.requestTransferSyntaxUID
+
+      if (requestTransferSyntaxUID) {
+        //Check TS is valid
+        if (!Object.keys(typeForTS).includes(requestTransferSyntaxUID)) {
```

Comment:
This is a slow way of doing things - just use if( !typeForTS[requestTransferSyntaxUID] ) is sufficient.